### PR TITLE
added dom task

### PR DESCRIPTION
**Pull request by chris-voitova**
DOM

[Demo](https://chris-voitova.github.io/kottans-dom/)
[Code base](https://github.com/chris-voitova/kottans-dom)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/chris-voitova/dom/script.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+const renderContent = (fromTarget) => {
+  const img = getImg(fromTarget);
+  content.innerHTML = "";
+  content.innerHTML += imgTemplate(img);
+};
+
+const handleChanges = () => {
+  list.addEventListener("click", ({ target }) => {
+    if (target.classList.contains("button")) {
+      [...list.children].forEach(({ children }) => {
```
Comment (MaxKalashnyk) [inline submissions/chris-voitova/dom/script.js:64]: You can just use `list.querySelector('.button.active').classList.remove("active")` here

**Review by MaxKalashnyk — CHANGES_REQUESTED** [review]
Good job!
As always the code is so clean and easy to follow. 
One minor suggestion and we're ready to merge this PR.

**Review by MaxKalashnyk — APPROVED** [review]
Well done! LGTM!

**Comment by chris-voitova** [comment]
> Good job!
> As always the code is so clean and easy to follow.
> One minor suggestion and we're ready to merge this PR.

It's very good news! Thanks for review! :)

**Pull request merged**